test/integration/recover_update_local_connection_test.go

```diff
@@ -47,6 +47,10 @@ func TestUpdateConnectionOnNSEChange(t *testing.T) {
 
 	logrus.Infof("Waiting for NSMD to update NSC connection")
 	k8s.WaitLogsContains(nsc, "monitoring-nsc", "Connection updated", defaultTimeout)
+
+	logrus.Info("Waiting to verify that NSC was not restarted")
```

Comment:
Let's move this logging into a new assertation function and also add a name of the target pod.